x-pack/legacy/plugins/siem/public/components/load_more_table/index.tsx

```diff
@@ -199,101 +183,85 @@ export class LoadMoreTable<T, U, V, W, X, Y, Z, AA, AB> extends React.PureCompon
+                        data-test-subj="loadingMoreSizeRowPopover"
+                        button={button}
+                        isOpen={this.state.isPopoverOpen}
+                        closePopover={this.closePopover}
+                        panelPaddingSize="none"
+                      >
+                        <EuiContextMenuPanel
+                          items={rowItems}
+                          data-test-subj="loadingMorePickSizeRow"
+                        />
+                      </EuiPopover>
+                    )}
+                  </EuiFlexItem>
 
-          {hasNextPage && (
-            <FooterAction>
-              <EuiFlexGroup
-                gutterSize="none"
-                alignItems="flexStart"
-                justifyContent="flexStart"
-                direction="row"
-              >
-                <EuiFlexItem grow={false}>
-                  {!isEmpty(itemsPerRow) && (
-                    <EuiPopover
-                      id="customizablePagination"
-                      data-test-subj="loadingMoreSizeRowPopover"
-                      button={button}
-                      isOpen={this.state.isPopoverOpen}
-                      closePopover={this.closePopover}
-                      panelPaddingSize="none"
+                  <EuiFlexItem grow={false}>
+                    <EuiButton
+                      data-test-subj="loadingMoreButton"
+                      isLoading={loading}
+                      onClick={this.props.loadMore}
+                      size="s"
                     >
-                      <EuiContextMenuPanel
-                        items={rowItems}
-                        data-test-subj="loadingMorePickSizeRow"
-                      />
-                    </EuiPopover>
-                  )}
-                </EuiFlexItem>
-                <EuiFlexItem>
-                  <EuiFlexGroup
-                    gutterSize="none"
-                    alignItems="flexStart"
-                    justifyContent="center"
-                    direction="row"
-                  >
-                    <EuiFlexItem grow={false}>
-                      <EuiButton
-                        data-test-subj="loadingMoreButton"
-                        isLoading={loading}
-                        onClick={this.props.loadMore}
-                      >
-                        {loading ? `${i18n.LOADING}` : i18n.LOAD_MORE}
-                      </EuiButton>
-                    </EuiFlexItem>
-                  </EuiFlexGroup>
-                </EuiFlexItem>
-              </EuiFlexGroup>
-            </FooterAction>
-          )}
-        </BasicTableContainer>
-      </EuiPanel>
+                      {loading ? `${i18n.LOADING}` : i18n.LOAD_MORE}
+                    </EuiButton>
+                  </EuiFlexItem>
+                </EuiFlexGroup>
+              </FooterAction>
+            )}
+
+            {loading && <Loader data-test-subj="LoadingPanelLoadMoreTable" overlay size="xl" />}
```

Comment:
nit: Your style usually looks to have `data-test-subj="` start with a lower case letter. Doesn't bother me though, but up to you if you want to keep them consistent.

Reply:
👍